### PR TITLE
Various fix-ups for the application volume adjustment feature

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
https://github.com/nvaccess/nvda/pull/16591#issuecomment-2327290160, https://github.com/nvaccess/nvda/pull/16591#issuecomment-2327326377

### Summary of the issue:
* The user guide is unclear on the drawbacks of enabling application volume adjustment.
* Various grammatical and stylistic issues.

### Description of how this pull request fixes the issue:
Updates what's new, the user guide, and various namings accordingly, per offline discussion with @mltony.

### Testing strategy:
Tested that feature works as expected. Alpha testing.

### Known issues with pull request:
None known

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

**Review by Qchristensen — APPROVED** [review]
Userguide change looks fine.

File: source/gui/settingsDialogs.py [hunk 1]
```diff
@@ -3028,8 +3028,9 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 
 		self._appendSoundSplitModesList(sHelper)
 
-		# Translators: This is a label for the applications volume adjuster combo box in settings.
-		label = _("&Application volume adjuster status")
+		# Translators: This is a label for the
+		# "allow application volume adjustment" combo box in settings.
+		label = _("&Allow application volume adjustment:")
```
Comment (CyrilleB79) [inline source/gui/settingsDialogs.py:3033]: A rewording suggestion to make this option clearer and in line with what you wrote in the User Guide.
* Indicate "NVDA" since it's the key point of this option: can NVDA play with the mixer?
* adjust -> modify: adjust seems suitable for volume commands but a bit less for mute command; modify seems a bit more general, thus suitable both for volume and mute commands.


```suggestion
		label = _("&Allow NVDA to modify the volume of other applications")
```

File: user_docs/en/userGuide.md [hunk 2]
```diff
@@ -2451,35 +2451,40 @@ By default only three modes are included.
...
 | Name | Key | Description |
 |---|---|---|
-| Increase applications volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
-| Decrease applications volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
+| Increase application volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
+| Decrease application volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
 
 This option is not available if you have started NVDA with [WASAPI disabled for audio output](#WASAPI) in Advanced Settings.
 
 ##### Muting other applications {#OtherAppMute}
```
Comment (CyrilleB79) [inline user_docs/en/userGuide.md:2484]: The option is called "Mute other apps" in the audio panel.

Could we reword it as well as the User Guide as follows:

```suggestion
##### Mute other applications {#OtherAppMute}
```

**Review by CyrilleB79 — COMMENTED** [review]
Thanks @codeofdusk for working to clarify this new feature. It's a key point to make this feature a real success.

Here is my first feedback; maybe there will be more comments after we have discussed it.

File: source/audio/appsVolume.py [hunk 3]
```diff
@@ -105,24 +105,30 @@ def _updateAppsVolumeImpl(
 	_activeCallback.register()
 
 
+WASAPI_DISABLED_MESSAGE: str = _(
+	# Translators: error message when wasapi is turned off.
+	"Application volumes cannot be modified by NVDA when WASAPI is disabled. "
```
Comment (seanbudd) [inline source/audio/appsVolume.py:110]: I don't think "volumes" makes sense here. typo?

```suggestion
	"Application volume cannot be modified by NVDA when WASAPI is disabled. "
```

File: source/config/featureFlagEnums.py [hunk 4]
```diff
@@ -70,10 +70,12 @@ class AppsVolumeAdjusterFlag(DisplayStringEnum):
...
-			# Translators: Label for applications volume adjuster in NVDA settings.
-			self.DISABLED: _("Disabled Applications volume adjuster"),
-			# Translators: Label for applications volume adjuster in NVDA settings.
-			self.ENABLED: _("Enabled Applications volume adjuster"),
+			# Translators: A choice disabling "application volume adjustment"
+			# in the audio settings panel.
+			self.DISABLED: _("No"),
+			# Translators: A choice enabling "application volume adjustment"
+			# in the audio settings panel.
+			self.ENABLED: _("Yes"),
```
Comment (seanbudd) [inline source/config/featureFlagEnums.py:78]: shouldn't this be disabled/enabled not no/yes? I think thats more common

File: source/gui/settingsDialogs.py [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
 # Rui Batista, Joseph Lee, Heiko Folkerts, Zahari Yurukov, Leonard de Ruijter,
-# Derek Riemer, Babbage B.V., Davy Kager, Ethan Holliger, Bill Dengler, Thomas Stivers,
-# Julien Cochuyt, Peter Vágner, Cyrille Bougot, Mesar Hameed, Łukasz Golonka, Aaron Cannon,
-# Adriani90, André-Abush Clause, Dawid Pieper, Heiko Folkerts, Takuya Nishimoto, Thomas Stivers,
-# jakubl7545, mltony, Rob Meredith, Burman's Computer and Education Ltd, hwf1324.
+# Derek Riemer, Babbage B.V., Davy Kager, Ethan Holliger, Bill Dengler,
+#  Thomas Stivers, Julien Cochuyt, Peter Vágner, Cyrille Bougot, Mesar Hameed,
+# Łukasz Golonka, Aaron Cannon, Adriani90, André-Abush Clause, Dawid Pieper,
+# Takuya Nishimoto, jakubl7545, Tony Malykh, Rob Meredith,
+# Burman's Computer and Education Ltd, hwf1324.
```
Comment (seanbudd) [inline source/gui/settingsDialogs.py:8]: why was this changed?

File: source/gui/settingsDialogs.py (hunk 1 above)
Comment (seanbudd) [reply source/gui/settingsDialogs.py:3034]: I think Cyrille's suggestion is simpler and most users won't make this implication. Thoughts @Qchristensen ?

Perhaps "set" might be clearer

```suggestion
		label = _("&Allow NVDA to setthe volume of other applications")
```

File: user_docs/en/changes.md [hunk 6]
```diff
@@ -6,9 +6,10 @@
 
 ### New Features
 
-* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. In order to use this feature, application volume adjuster needs to be enabled in Audio pane of NVDA settings. (#16052, @mltony)
-* Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`.
-In order to use this feature, the application volume adjuster needs to be enabled in the Audio category of NVDA settings. (#16052, @mltony)
+* Commands to adjust the volume of other applications besides NVDA have been added. To use this feature, "allow application volume adjustment" must be enabled in the audio settings panel. (#16052, @mltony, @codeofdusk)
+    * <kbd>NVDA</kbd>+<kbd>Alt</kbd>+<kbd>Pageup</kbd>: Increase the volume of all other applications.
```
[the remark this answers is not in the document]
Comment (seanbudd) [reply user_docs/en/changes.md:10]: We use code fortmatting for these currently. If you'd like to propose a change to our standards, please open a new issue or PR to userGuideStandards.ms

File: user_docs/en/changes.md [hunk 7]
```diff
@@ -6,9 +6,10 @@
 
 ### New Features
 
-* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. In order to use this feature, application volume adjuster needs to be enabled in Audio pane of NVDA settings. (#16052, @mltony)
-* Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`.
-In order to use this feature, the application volume adjuster needs to be enabled in the Audio category of NVDA settings. (#16052, @mltony)
+* Commands to adjust the volume of other applications besides NVDA have been added. To use this feature, "allow application volume adjustment" must be enabled in the audio settings panel. (#16052, @mltony, @codeofdusk)
```
Comment (seanbudd) [inline user_docs/en/changes.md:9]:
```suggestion
* Commands to adjust the volume of other applications besides NVDA have been added.
To use this feature, "allow application volume adjustment" must be enabled in the audio settings panel. (#16052, @mltony, @codeofdusk)
```

File: user_docs/en/userGuide.md [hunk 8]
```diff
@@ -2451,35 +2451,38 @@ By default only three modes are included.
...
 This slider allows you to adjust the volume of all currently running applications other than NVDA.
-This volume setting will apply to all other applications sound output, even if they start after this setting is changed.
 This volume can also be controlled via the following keyboard commands from anywhere:
 
 | Name | Key | Description |
 |---|---|---|
-| Increase applications volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
-| Decrease applications volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
+| Increase application volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
+| Decrease application volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
```
Comment (seanbudd) [inline user_docs/en/userGuide.md:2479]: I think plural makes sense here, this is adjust the volume of all other apps

File: source/config/featureFlagEnums.py (hunk 4 above)
Comment (CyrilleB79) [reply source/config/featureFlagEnums.py:78]: @seanbudd, if the option name is "Allow NVDA to modify the volume of other applications" or "Allow whatever"), Yes/No makes much more sense than Enabled/Disabled which is a bit confusing.

Once, and if, you agree with Yes/No, other things need to be changed in the code: variable names, other strings, etc. We need to double check that all is consistent with Yes/No.

File: source/gui/settingsDialogs.py (hunk 1 above)
Comment (CyrilleB79) [reply source/gui/settingsDialogs.py:3034]: My impression is that "modify" seems more suitable for both the volume change and the mute command, whereas with "set" it seems less obvious that it also impacts the mute command.

It's only the personal opinion of a non native English speaker, so feel free to choose "set" or another wording.

Comment (XLTechie) [reply source/gui/settingsDialogs.py:3034]: Personally, I would prefer modify as Cyrille suggested, or the original adjust, over set.
Or possibly "control".

```suggestion
		label = _("&Allow NVDA to control the volume of other applications:")
```

I do understand where @codeofdusk is coming from with audio ducking. I'm not sure users will make that assumption, though this point should be clarified somewhere if it isn't already.

File: source/gui/settingsDialogs.py (hunk 5 above)
Comment (codeofdusk) [reply source/gui/settingsDialogs.py:8]: I removed duplicate names and fixed attributions.

File: source/config/featureFlagEnums.py (hunk 4 above)
Comment (codeofdusk) [reply source/config/featureFlagEnums.py:78]: This should be `no`/`yes` to match "allow"

File: user_docs/en/changes.md [hunk 9]
```diff
@@ -6,10 +6,11 @@
...
 
-* Commands to adjust the volume of other applications besides NVDA have been added. To use this feature, "allow application volume adjustment" must be enabled in the audio settings panel. (#16052, @mltony, @codeofdusk)
-    * <kbd>NVDA</kbd>+<kbd>Alt</kbd>+<kbd>Pageup</kbd>: Increase the volume of all other applications.
-    * <kbd>NVDA</kbd>+<kbd>Alt</kbd>+<kbd>Pagedown</kbd>: Decrease the volume of all other applications.
-    * <kbd>NVDA</kbd>+<kbd>Alt</kbd>+<kbd>Delete</kbd>: Mute the volume of all other applications.
+* Commands to adjust the volume of other applications besides NVDA have been added.
+To use this feature, "allow NVDA to control the volume of other applications" must be enabled in the audio settings panel. (#16052, @mltony, @codeofdusk)
+    * `NVDA`+`Alt`+`Pageup`: Increase the volume of all other applications.
+    * `NVDA`+`Alt`+`Pagedown`: Decrease the volume of all other applications.
+    * `NVDA`+`Alt`+`Delete`: Mute the volume of all other applications.
```
Comment (seanbudd) [inline user_docs/en/changes.md:13]:
```suggestion
    * `NVDA+alt+pageUp`: Increase the volume of all other applications.
    * `NVDA+alt+pageDown`: Decrease the volume of all other applications.
    * `NVDA+alt+delete`: Mute the volume of all other applications.
```

File: user_docs/en/userGuide.md [hunk 10]
```diff
@@ -2475,8 +2476,8 @@ This volume can also be controlled via the following keyboard commands from anyw
 
 | Name | Key | Description |
 |---|---|---|
-| Increase application volume | `NVDA+alt+pageUp` | Increases volume of all applications except NVDA. |
-| Decrease application volume | `NVDA+alt+pageDown` | Decreases volume of all applications except NVDA. |
+| Increase application volume | `NVDA+alt+pageUp` | Increases the volume of all applications except NVDA. |
+| Decrease application volume | `NVDA+alt+pageDown` | Decreases the volume of all applications except NVDA. |
```
Comment (seanbudd) [inline user_docs/en/userGuide.md:2480]:
```suggestion
| Increase applications volume | `NVDA+alt+pageUp` | Increases the volume of all applications except NVDA. |
| Decrease applications volume | `NVDA+alt+pageDown` | Decreases the volume of all applications except NVDA. |
```

Comment (codeofdusk) [reply user_docs/en/userGuide.md:2480]: "applications volume" doesn't sound right to me. How about increase/decrease volume of other applications?

Comment (seanbudd) [reply user_docs/en/userGuide.md]: That would be fine too, I just think the current writing implies it only changes the current applications volume

**Review by seanbudd — APPROVED** [review]

**Comment by seanbudd** [comment]
Thanks @codeofdusk !
